src/NovelRT/ResourceManagement/Desktop/DesktopResourceLoader.cpp

```diff
@@ -294,15 +296,15 @@ namespace NovelRT::ResourceManagement::Desktop
     {
         filePath = _resourcesRootDirectory / filePath;
 
-        jsoncons::json j(jsoncons::json_object_arg);
-        std::vector<jsoncons::json> memberMetadataJson{};
+        nlohmann::json j{};
+        nlohmann::json memberMetadataJson{};
 
         for (auto&& member : package.memberMetadata)
         {
-            jsoncons::json newMemberJson(jsoncons::json_object_arg);
+            nlohmann::json newMemberJson{};
 
             newMemberJson["name"] = member.name;
-            newMemberJson["type"] = static_cast<uint32_t>(member.type);
+            newMemberJson["type"] = member.type;
```

Comment:
Is `member.type` already a uint32_t for nlohmann::json?

Reply:
peeking at the [json header file](https://github.com/nlohmann/json/blob/develop/include/nlohmann/detail/conversions/to_json.hpp#L320), it does seem to already look at the underlying value in the enum, but i could be wrong. it is not the easiest library to look through

Reply:
It's converted to an int by default, but it can be customised: https://json.nlohmann.me/features/enum_conversion/

I would suggest doing so, as this avoids potential reordering errors.